shims/common/src/main/scala/io/glutenproject/GlutenConfig.scala

```diff
@@ -395,6 +395,12 @@ object GlutenConfig {
       k => {
         if (conf.contains(k)) {
           nativeConfMap.put(k, conf(k))
+        } else {
+          // Session's local time zone must be set. If not explicitly set by user,
+          // its default value is used, consistent with spark.
+          if (k.equals(SQLConf.SESSION_LOCAL_TIMEZONE.key)) {
+            nativeConfMap.put(k, SQLConf.SESSION_LOCAL_TIMEZONE.defaultValueString)
+          }
```

Comment:
Probably can be placed to `GlutenDriverPlugin#setPredefinedConfigs`.

One of the reason is that the default time zone may vary from executor to driver, I assume the one from driver should be chosen? Here we use executor's time zone.

Reply:
Hi @zhztheplayer, thanks for your comment!
Yes, it may be possible that driver/executors are in different default time zone due to the different time zone config for OS. And it is better to have a unique timezone setting from driver side. Just updated this part. Thanks!